### ISSUE TITLE
[REVIEW]: The SAGE Rejected Article Tracker

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#3348]
**Submitting author:** @ad48 (<a href="http://orcid.org/0000-0002-8529-9990">Adam Day</a>)
**Repository:** <a href="https://github.com/sagepublishing/rejected_article_tracker_pkg" target ="_blank">https://github.com/sagepublishing/rejected_article_tracker_pkg</a>
**Version:** 1.5.8
**Editor:** @danielskatz
**Reviewer:** @mfenner, @dhimmel
**Archive:** <a href="https://doi.org/10.5281/zenodo.5155973" target="_blank">10.5281/zenodo.5155973</a>

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/92ca2de4a5f90d9250840dfc512a4855/status.svg)](https://joss.theoj.org/papers/92ca2de4a5f90d9250840dfc512a4855)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/92ca2de4a5f90d9250840dfc512a4855"><img src="https://joss.theoj.org/papers/92ca2de4a5f90d9250840dfc512a4855/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/92ca2de4a5f90d9250840dfc512a4855/status.svg)](https://joss.theoj.org/papers/92ca2de4a5f90d9250840dfc512a4855)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/sagepublishing/rejected_article_tracker_pkg" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@mfenner & @dhimmel, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @danielskatz know.

✨ **Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest** ✨ 

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/sagepublishing/rejected_article_tracker_pkg">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@ad48) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Does the paper have a section titled 'Statement of Need' that clearly states what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] #3466

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/sagepublishing/rejected_article_tracker_pkg">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@ad48) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Does the paper have a section titled 'Statement of Need' that clearly states what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @mfenner, @dhimmel it looks like you're currently assigned to review this paper :tada:.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.88  T=0.07 s (1043.1 files/s, 63974.8 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Python                          62            571            518           2233
Markdown                         5            102              0            349
TeX                              1              5              0            164
Jupyter Notebook                 1              0            306             85
YAML                             1              7              0             41
Bourne Shell                     2              4             21             10
-------------------------------------------------------------------------------
SUM:                            72            689            845           2882
-------------------------------------------------------------------------------


Statistical information for the repository 'e94253f5402812f1c94bde02' was
gathered on 2021/06/09.
The following historical commit information, by author, was found:

Author                     Commits    Insertions      Deletions    % of changes
Ad48                             3             5              3            0.20
Adam Day                        32          2446            171           65.00
Andy Hails                      21          1159            137           32.19
ad48                             5            45             27            1.79
aday                             4            19             12            0.77
andyleanlibrary                  1             1              1            0.05

Below are the number of rows from each author that have survived and are still
intact in the current revision:

Author                     Rows      Stability          Age       % in comments
Adam Day                   2338           95.6          4.0               13.05
Andy Hails                  915           78.9          1.9                2.62
ad48                         49          108.9          0.5                6.12
aday                         19          100.0          2.8                5.26
andyleanlibrary               1          100.0          3.5                0.00
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1002/bjs.6880 is OK
- 10.1007/s00134-018-5407-2 is OK
- 10.3138/W706-5884-KG84-J282 is OK
- 10.1111/anae.13829 is OK
- 10.23876/j.krcp.20.392 is OK
- 10.1177/0011392116663807 is OK
- 10.1093/ije/dyz187 is OK
- 10.1186/s41073-017-0031-1 is OK
- 10.1371/journal.pbio.3000959 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: :point_left:

**Comment by danielskatz** [comment]
@mfenner and @dhimmel - Thanks for agreeing to review this submission.
This is the review thread for the paper. All of our communications will happen here from now on.

Both reviewers have checklists at the top of this thread with the JOSS requirements. As you go over the submission, please check any items that you feel have been satisfied. There are also links to the JOSS reviewer guidelines.

Please read the first couple of comments in this issue carefully, so that you can accept the invitation from JOSS and be able to check items, and so that you don't get overwhelmed with notifications from other activities in JOSS.

The JOSS review is different from most other journals. Our goal is to work with the authors to help them meet our criteria instead of merely passing judgment on the submission. As such, reviewers are encouraged to submit issues and pull requests on the software repository. When doing so, please mention `openjournals/joss-reviews#3348` so that a link is created to this thread (and I can keep an eye on what is happening). Please also feel free to comment and ask questions on this thread. In my experience, it is better to post comments/questions/suggestions as you come across them instead of waiting until you've reviewed the entire package.

We aim for reviews to be completed within about 2-4 weeks. Please let me know if either of you require some more time. We can also use Whedon (our bot) to set automatic reminders if you know you'll be away for a known period of time.

Please feel free to ping me (@danielskatz) if you have any questions/concerns.

**Comment by danielskatz** [comment]
@dhimmel - just a ping that if you can accept the reviewer invitation in the first comment in this issue, that would be great - it will eventually expire which will make it harder for you to start the review (when you are ready)

**Comment by dhimmel** [comment]
# Daniel Himmelstein Review

Draft: will keep updating this comment.

Here's a list of issues related to my review:

- https://github.com/sagepublishing/rejected_article_tracker_pkg/issues/4

**Comment by ad48** [comment]
Thank you for your comments. My co-author and I will discuss these and make amendments.

**Comment by danielskatz** [comment]
@ad48  - any news?

**Comment by danielskatz** [comment]
@dhimmel - I'm going to tag your issues with this issue, which helps me see here if they are still open or closed

**Comment by danielskatz** [comment]
> @ad48 - any news?

sorry, I now see you are working on the issue - nvm...

**Comment by whedon** [comment]
:wave: @mfenner, please update us on how your review is going (this is an automated reminder).

**Comment by whedon** [comment]
:wave: @dhimmel, please update us on how your review is going (this is an automated reminder).

**Comment by danielskatz** [comment]
👋 @mfenner - it would be great if you can get started when you get the chance

**Comment by danielskatz** [comment]
👋 @ad48 - any update on the issues from @dhimmel?

**Comment by ad48** [comment]
@danielskatz my coauthor and I have been discussing it and we've made a few changes. We resolved all but 1 of the issues. (Should I close those ones now?)

**Comment by mfenner** [comment]
@danielskatz will work on this by tomorrow.

**Comment by dhimmel** [comment]
> we've made a few changes

Ah I hadn't seen https://github.com/sagepublishing/rejected_article_tracker_pkg/commit/c30b755619bf0e9517f848827911797de5e37275 with changes tot paper, will take a look at that and update the issues.

**Comment by dhimmel** [comment]
Went through my open issues and closed the resolved ones. Still one remaining.

**Comment by ad48** [comment]
I've just pushed some changes, but I have a few more things to change in the paper. Will try and get those done tomorrow.

**Comment by danielskatz** [comment]
👋 @mfenner - I see you started, but haven't made much progress in the checkist - is there any update?

**Comment by ad48** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: :point_left:

**Comment by ad48** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: :point_left:

**Comment by danielskatz** [comment]
👋 @ad48 - how are you doing in responding to the issues from the reviewers?  Can we get an update here?

**Comment by ad48** [comment]
@danielskatz  We have 2 things left on the todo list: one is to create a sample file to make testing the app easy. The other is to post the training data to an open data repository. Both are quite do-able, but time has been a little limited over this last week. Hopefully will have it done this week. The feedback from both referees has been highly constructive and greatly appreciated.

**Comment by danielskatz** [comment]
@ad48 - any news on this?

**Comment by ad48** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: :point_left:

**Comment by ad48** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: :point_left:

**Comment by ad48** [comment]
@danielskatz I think that's all of the issues responded to now. Many thanks.

**Comment by danielskatz** [comment]
👋 @dhimmel and @mfenner - Can you take a look at this again and see if the changes address your points, and if you can now check all/additional review criteria?

**Comment by dhimmel** [comment]
I'm waiting on a new PyPI release as noted at https://github.com/sagepublishing/rejected_article_tracker_pkg/issues/9#issuecomment-881663829

**Comment by danielskatz** [comment]
Should we reopen that issue?

**Comment by ad48** [comment]
I've pushed the latest version to pypi now. Running `pip install rejected_article_tracker` will install version 1.5.6, then `python run_sample.py` produces an `output.xlsx` file which demonstrates the process working. 

For context: 
- The output file actually shows that the tracker fails to find one of the example articles. I thought that was worth leaving in so that users know what a null result looks like. 
- We don't typically share data on real rejected articles because there are some data-protection issues in doing so.  The examples are all fake - they are real papers that are very highly cited and therefore unlikely to have been rejected from anywhere.

**Comment by danielskatz** [comment]
👋 @dhimmel - I hope you can now move forward on the remaining items?

**Comment by danielskatz** [comment]
👋 @mfenner - Can you remind us what your status is as well?

**Comment by mfenner** [comment]
All my issues have been adequately addressed.

**Comment by danielskatz** [comment]
> All my issues have been adequately addressed.

Can you check off the other 4 items in your review then?

**Comment by mfenner** [comment]
> Can you check off the other 4 items in your review then?

Done. Let me know if there is anything else you need from me.

**Comment by danielskatz** [comment]
Thanks @mfenner - I'll assume from this and your messages that you are good with this going forward to acceptance.

**Comment by mfenner** [comment]
Yes, the authors have addressed all my questions and I am good with accepting the manuscript.

**Comment by danielskatz** [comment]
👋 @dhimmel - Any update from you on the remaining items?

**Comment by dhimmel** [comment]
Still waiting to check off the following bullet:

>  Functionality: Have the functional claims of the software been confirmed?

Opened https://github.com/sagepublishing/rejected_article_tracker_pkg/issues/14 recently when the README code failed.

**Comment by dhimmel** [comment]
The authors have addressed all of the issues I opened. No more objections on my end to accepting this manuscript!

**Comment by danielskatz** [comment]
@dhimmel - What about the two items in your checklist that are not checked?

**Comment by dhimmel** [comment]
> What about the two items in your checklist that are not checked?

Looked over the guidance for "Substantial scholarly effort" and "Community guidelines" and decided that I can check these off.

**Comment by danielskatz** [comment]
Great - thanks!

**Comment by danielskatz** [comment]
@whedon check references

**Comment by danielskatz** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1002/bjs.6880 is OK
- 10.1007/s00134-018-5407-2 is OK
- 10.3138/W706-5884-KG84-J282 is OK
- 10.1111/anae.13829 is OK
- 10.23876/j.krcp.20.392 is OK
- 10.1177/0011392116663807 is OK
- 10.1093/ije/dyz187 is OK
- 10.1186/s41073-017-0031-1 is OK
- 10.1371/journal.pbio.3000959 is OK
- 10.1007/s11192-021-03900-7 is OK
- 10.1080/02763869.2020.1704597 is OK
- 10.5281/zenodo.5122848 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: :point_left:

**Comment by danielskatz** [comment]
@ad48 - I've suggested some changes in https://github.com/sagepublishing/rejected_article_tracker_pkg/pull/15.  In addition, the table that follows line 73 overruns the margin - perhaps you can shorten some of the headers or make them two lines to avoid this. Also, the reference to Incorvia2015 seems incomplete.

**Comment by ad48** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: :point_left:

**Comment by ad48** [comment]
@whedon generate pdf

**Comment by ad48** [comment]
@whedon check references

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: :point_left:

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1002/bjs.6880 is OK
- 10.1007/s00134-018-5407-2 is OK
- 10.3138/W706-5884-KG84-J282 is OK
- 10.1111/anae.13829 is OK
- 10.23876/j.krcp.20.392 is OK
- 10.1177/0011392116663807 is OK
- 10.1093/ije/dyz187 is OK
- 10.1186/s41073-017-0031-1 is OK
- 10.1371/journal.pbio.3000959 is OK
- 10.1007/s11192-021-03900-7 is OK
- 10.1080/02763869.2020.1704597 is OK
- 10.5281/zenodo.5122848 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by ad48** [comment]
@danielskatz I have tried adding linebreaks to the column headings, but it seems whedon couldn't render those. So I have changed the headings to say 'ArXiv' and 'VOR' with some more explanatory text about what VOR means in the preceding paragraph. I think this gets around the problem with overrunning the margins. 

I also removed the 'Incorvia2015' reference. That was supposed to reference the patent for the HighWire tool in the other reference. Thinking about it, the patent isn't particularly relevant given the context of the reference which is only pointing out that commercial alternatives exist. In the recent weeks (since we submitted this paper), Dimensions have started promoting their rejected article tracking service. So we have added a link to that since it's another relevant example.

@whedon generate pdf

**Comment by ad48** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: :point_left:

**Comment by danielskatz** [comment]
@whedon recommend-accept

**Comment by whedon** [comment]
No archive DOI set. Exiting...

**Comment by danielskatz** [comment]
Thanks - it all looks good to me now.  At this point could you:
- [ ] Make a tagged release of your software, and list the version tag of the archived version here.
- [ ] Archive the reviewed software in Zenodo or a similar service (e.g., figshare, an institutional repository)
- [ ] Check the archival deposit (e.g., in Zenodo) has the correct metadata. This includes the title (should match the paper title) and author list (make sure the list is correct and people who only made a small fix are not on it). You may also add the authors' ORCID.
- [ ] Please list the DOI of the archived version here.

I can then move forward with accepting the submission.

**Comment by ad48** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03348/joss.03348/10.21105.joss.03348.pdf) :page_facing_up: :point_left:

**Comment by ad48** [comment]
- @danielskatz @mfenner @dhimmel I have made a minor amendment to the acknowledgements section to include all of you. (I assume that's OK - I couldn't find guidelines on this.) I wanted to pass on our thanks to you for all of your help with this. 
- Tagged release 1.5.8 
- Archived in Zenodo
- Metadata appears to be correct (including ORCIDs)
- Zenodo DOI: 10.5281/zenodo.5155973

**Comment by danielskatz** [comment]
I'm ok with being listed there, but I'm the editor, not a reviewer.

**Comment by danielskatz** [comment]
@whedon set 1.5.8 as version

**Comment by whedon** [comment]
OK. 1.5.8 is the version.

**Comment by danielskatz** [comment]
@whedon set 10.5281/zenodo.5155973 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.5155973" target="_blank">10.5281/zenodo.5155973</a> is the archive.

**Comment by danielskatz** [comment]
If you can change the ack to list me as editor rather than reviewer, please do - you don't need to create a new release or update the archive, as we don't even require the paper to be in the archive, but if you want to, you can.  Let me know either way, and I'll move on the next step.

**Comment by ad48** [comment]
I've updated the acknowledgement to include roles. Should I re-generate the pdf?

**Comment by danielskatz** [comment]
No, I will do this as part of the next step

**Comment by danielskatz** [comment]
@whedon recommend-accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1002/bjs.6880 is OK
- 10.1007/s00134-018-5407-2 is OK
- 10.3138/W706-5884-KG84-J282 is OK
- 10.1111/anae.13829 is OK
- 10.23876/j.krcp.20.392 is OK
- 10.1177/0011392116663807 is OK
- 10.1093/ije/dyz187 is OK
- 10.1186/s41073-017-0031-1 is OK
- 10.1371/journal.pbio.3000959 is OK
- 10.1007/s11192-021-03900-7 is OK
- 10.1080/02763869.2020.1704597 is OK
- 10.5281/zenodo.5122848 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/2490

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/2490, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by danielskatz** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1422596755296866305) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/2491
1. Wait a couple of minutes, then verify that the paper DOI resolves [https://doi.org/10.21105/joss.03348](https://doi.org/10.21105/joss.03348)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? Notify your editorial technical team...

**Comment by danielskatz** [comment]
Congratulations @ad48 (Adam Day) and Andrew Hails!!

And thanks to @mfenner and @dhimmel for reviewing!

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.03348/status.svg)](https://doi.org/10.21105/joss.03348)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.03348">
  <img src="https://joss.theoj.org/papers/10.21105/joss.03348/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.03348/status.svg
   :target: https://doi.org/10.21105/joss.03348
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.03348/status.svg)](https://doi.org/10.21105/joss.03348)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss

**Comment by ad48** [comment]
This is brilliant news! Thank you everyone for all your help :)
